git/githistory/rewriter.go

```diff
@@ -253,6 +267,12 @@ func (r *Rewriter) Rewrite(opt *RewriteOptions) ([]byte, error) {
 			if err != nil {
 				return nil, err
 			}
+			if objectMapFile != nil {
+				_, err := fmt.Fprintf(objectMapFile, "%s,%s\n", hex.EncodeToString(oid), hex.EncodeToString(newSha))
```

Comment:
This line can be shortened by using the `%x` formatting verb, which performs the `hex.EncodeToString` call transparently:

```go
_, err := fmt.Fprintf(objectMapFile, "%x,%x\n", oid, newSha))
```

is equivalent to:

```go
_, err := fmt.Fprintf(objectMapFile, "%s,%s\n", hex.EncodeToString(oid), hex.EncodeToString(newSha))
```

##

Since the line is shorter, do you think we can compact it even further by:

```
if _, err := fmt.Fprintf(objectMapFile, "%x,%x\n", oid, newSha)); err != nil {
        return nil, err
}
```

Reply:
I have made the change.